roles/satellite-clone/defaults/main.yml

```diff
@@ -1,6 +1,6 @@
 ---
 # more info about these variables can be found in satellite-clone-vars.sample.yml in the root directory of this project
-satellite_version: 6.2
+satellite_version: 6.1
```

Comment:
I am questioning the need for this variable in the defaults/main.yml file. You can remove this variable from the defaults/main.yml file and set 6.1 or 6.2 in the code itself.